home/templates/home/prerelease/session_list.html

```diff
@@ -25,7 +25,17 @@ <h1 class="text-5xl">Sessions</h1>
         </div>
         <div class="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 pt-4">
             {% for session in sessions %}
+                {% if session.is_accepting_applications %}
                 {% include 'home/includes/session_card.html' %}
+                {% endif %}
+                {% if forloop.last and not session.is_accepting_applications %}
```

Comment:
I agree with your idea of checking to see if the end_date is in the future. We only want to show this message if we have no session in the future.

Reply:
I'm not sure. How can I check if there's no session in the future? Is it ok to check the end of the program? For example for session2 the end is 12th August, if there are no other sessions available on 13th August then I display the placeholder, is it ok?

Reply:
Yes, you're correct. We would make the assumption that the first session will be the newest (the view needs to be updated to do this). It could potentially even be in the future. Or it could be in the past. When it's in the past, that's when we want to show the placeholder.